bin/benchmark_pydy_code_gen.py

```diff
@@ -45,11 +40,8 @@ def run_benchmark(max_num_links, num_time_steps=1000):
         for i in range(n):
             parameter_vals += [arm_length, bob_mass]
 
-        # odeint arguments
-        x0 = hstack((0, pi / 2 * ones(len(results[3]) - 1), 1e-3 *
-                    ones(len(results[4]))))
-        args = {'constants': dict(zip(results[2], array(parameter_vals)))}
```

Comment:
I want to ask one thing, here x0  is an array but while setting initial conditions of a system it checks if its a dict and whether it matches its states to floats i.e. initial_conditions.keys(). Do both of them works with odeint? I have used zip() with x0 and sys.states to create a dict, is it correct?